libs/base/Data/Either.idr

```diff
@@ -41,6 +41,11 @@ export
 Uninhabited (IsRight (Left x)) where
   uninhabited ItIsRight impossible
 
+||| Returns the `r` value of an `Either l r` which is proved `Right`.
+fromRight : (e : Either l r) -> {auto 0 isRight : IsRight e} -> r
```

Comment:
`auto` should be removed to avoid unpredictable search results and to match the existing API of `fromJust` https://github.com/idris-lang/Idris2/blob/be2ec7d1fd8897ea0b786de08dd366089e483d29/libs/base/Data/Maybe.idr#L43